### PR TITLE
Update docs: Ninja now required to build with MSVC

### PR DESCRIPTION
**Pull request by djcsdy**
Hi!

Thank you for merging #47 😄.

This PR updates the docs to take into account the changes made by that PR.

Thanks!

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -26,9 +26,9 @@ install:
   - if not %target% == %host% rustup target add %target%
   - rustc -vV && cargo -vV
 
-  # Because that MSBUild does not support path longer than MAX_PATH
-  # (https://github.com/Microsoft/msbuild/issues/53),
-  # we use ninja to build.
+  # Ninja is required to build with Visual Studio because MSBuild does not
+  # support pathnames longer than MAX_PATH
```
Comment (antiagainst) [inline .appveyor.yml]: paths

**Review by antiagainst — COMMENTED** [review]
Thanks for the doc update! Could you also update https://github.com/google/shaderc-rs/blob/master/src/lib.rs#L31?

**Comment by knappador** [comment]
Merge hazard on #46.  Looks compatible-ish

**Comment by antiagainst** [comment]
Merged. Thanks! :)

**Pull request merged**